apps/aecore/src/aec_block_candidate.erl

```diff
@@ -8,17 +8,27 @@
 
 -export([ apply_block_txs/5
         , apply_block_txs_strict/5
-        , calculate_fee/1
-        , calculate_total_fee/1
         , create/1
         , create_with_state/4
+        , update/3
         ]).
 
+-ifdef(TEST).
 -export([adjust_target/2]).
+-endif.
+
+-export_type([block_info/0]).
+
+-include("blocks.hrl").
+
+-opaque block_info() :: #{trees := aec_trees:trees(),
+                          tot_fee := non_neg_integer(),
```

Comment:
`tot_fee` is total fee?

Reply:
Yes

Reply:
This is refactoring. I plan to change this as https://github.com/aeternity/epoch/pull/1230/commits/4a8cdddd30d16aedb3437510fe1155efd4e4587b (separate PR).